packages/babel-plugin/src/styled/index.tsx

```diff
@@ -4,14 +4,22 @@ import { buildStyledComponent } from '../utils/ast-builders';
 import { buildCss } from '../utils/css-builders';
 import { Metadata, Tag } from '../types';
 
+interface StyledData {
+  tag: {
```

Comment:
This is not type `Tag` now which is coming from `../types`? Got confused 😅

Reply:
Oh I just extracted the types from the return types. I'll see if they are the same 🤔